File: golang/mongo.go
package main

import(
    "https://github.com/geekjr/translate-api-go"
	"go.mongodb.org/mongo-driver/bson"
	"go.mongodb.org/mongo-driver/mongo"
	"go.mongodb.org/mongo-driver/mongo/options"
	"go.mongodb.org/mongo-driver/mongo/readpref"
)

func main() {
	uri := "MONGO-CONNECTION-STRING"
	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
	client, err := mongo.Connect(ctx, options.Client().ApplyURI(uri))
	collection := client.Database("DATABASE").Collection("COLLECTION")
	cursor, err := collection.Find(context.TODO(), bson.D{})
	for doc in cursor {
        json = translateapi.JsonRequest("YOUR-API-KEY", "%d", doc)
    }
}

File: javascript/mongo.js
import JsonRequest from "translation-api-js-client";

let json
const url = "CONNECTIOn STRING";
const client = new MongoClient(url);
await client.connect();
const db = client.db("DATABASE TO USE");
const col = db.collection("COLLECTION TO USE");
const cursor = col.find();
await cursor.forEach(json = JsonRequest({"THE": "JSON", "DOC": "TO TRANSLATE"}, "YOUR-API-KEY"), console.log(json));